discord/enums.py

```diff
@@ -347,6 +350,10 @@ class AuditLogAction(Enum):
     thread_update                 = 111
     thread_delete                 = 112
     app_command_permission_update = 121
+    auto_moderation_rule_create   = 140
+    auto_moderation_rule_update   = 141
+    auto_moderation_rule_delete   = 142
+    auto_moderation_block_message = 143
```

Comment:
Maybe these should be shortened for consistency.

```suggestion
    automod_rule_create           = 140
    automod_rule_update           = 141
    automod_rule_delete           = 142
    automod_block_message         = 143
```